fv_be/app/models/dictionary.py

```diff
@@ -0,0 +1,153 @@
+from django.db import models
+from django.utils.translation import gettext as _
+
+# FirstVoices
+from .base import BaseModel
+from .category import Category
+from .part_of_speech import PartOfSpeech
+
+
+class Note(BaseModel):
+    """Model for notes associated to each dictionary entry."""
+
+    # from fv:notes,fv:general_note, fv:cultural_note, fv:literal_translation, fv-word:notes, fv-phrase:notes
+    text = models.TextField()
+    dictionary_entry = models.ForeignKey(
+        "DictionaryEntry", on_delete=models.CASCADE, related_name="notes"
+    )
+
+    def __str__(self):
+        return self.text
+
+
+class Acknowledgement(BaseModel):
```

Comment:
Same comment as the Notes model applies here. I'm ok with keeping it as-is, just thought I'd mention it.